x-pack/plugins/lens/public/drag_drop/providers.tsx

```diff
@@ -176,7 +258,12 @@ export function ReorderProvider({
             </p>
             <p id={`lnsDragDrop-reorderInstructions-${id}`}>
               {i18n.translate('xpack.lens.dragDrop.reorderInstructions', {
-                defaultMessage: `Press space bar to start a drag. When dragging, use arrow keys to reorder. Press space bar again to finish.`,
+                defaultMessage: `Press space bar to start reordering the dimension group. When dragging, use arrow keys to reorder. Press space bar again to finish.`,
```

Comment:
It seems like this action is attached to a button's click event and not to a `space` keyboard event. In that case, buttons will react to both `space` and `enter` (and some assistive tech can customize this further). 

Instead of telling the user to press a specific button, we can use the word "activate" to include whatever it is they might do. 

```suggestion
                defaultMessage: `Activate this button to start reordering the dimension group. Use arrow keys to reorder. Activate this button again to finish.`,
```

Reply:
It is attached to `Enter` or `Space` and I couldn't make it work with click (comment [below](https://github.com/elastic/kibana/pull/88578#discussion_r565085737)) – do you still this change makes sense even if we don't make it onClick?